fetchDependencies

```diff
@@ -444,6 +452,19 @@ EXT_DEPS=ExternalDependencies
 XC_PROJ="${EXT_DEPS}.xcodeproj"
 XC_DD_PATH="${EXT_DIR}/build"
 
+# Determine if xcpretty is present
+XCPRETTY_PATH=$(command -v xcpretty 2> /dev/null)
```

Comment:
On my local machine, `fetchDependencies` doesn't get past this line. No error reported...even if I remove `2> /dev/null`.